core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -725,6 +725,7 @@ export class LoggedInUser extends BaseUser {
         'The dismiss button does not close the Thanks for Donating popup!'
       );
     }
+    await this.page.waitForSelector(donatePage);
```

Comment:
After clicking the Dismiss button, can we also try waiting for the "Thanks for donating" pop-up to disappear using `await this.page.waitForSelector(thanksForDonatingClass, { hidden: true });` before checking if it disappeared?

Reply:
yup, would be a good add one. 
Thanks.